orca-core/src/main/groovy/com/netflix/spinnaker/orca/pipeline/persistence/jedis/Migrator.groovy

```diff
@@ -0,0 +1,50 @@
+package com.netflix.spinnaker.orca.pipeline.persistence.jedis
+
+import com.netflix.spinnaker.orca.ExecutionStatus
+import com.netflix.spinnaker.orca.pipeline.model.Execution
+import com.netflix.spinnaker.orca.pipeline.persistence.ExecutionRepository
+import groovy.transform.CompileDynamic
+import groovy.transform.CompileStatic
+import org.springframework.beans.factory.annotation.Autowired
+import org.springframework.context.ApplicationListener
+import org.springframework.context.event.ContextRefreshedEvent
+import org.springframework.stereotype.Component
+import redis.clients.jedis.Jedis
+import redis.clients.util.Pool
+import rx.Observable
+
+@Component
+@CompileStatic
+class Migrator implements ApplicationListener<ContextRefreshedEvent> {
```

Comment:
This looks old, but possible adaptable to a point-in-time migration?

Given the OSS ramifications, we could perhaps get this done quickly and not worry about releasing a migrator.

Alternatively, we have something that runs on startup or something and attempts to perform a migration, and it lives in the repo for some period of time.